.env.example

```diff
@@ -0,0 +1,14 @@
+AZURE_CLIENT_ID=
+AZURE_TENANT_ID=
+
+AZURE_OPENAI_EMBEDDING_DEPLOYMENT_NAME=text-embedding-ada-002
+AZURE_OPENAI_EMBEDDING_MODEL_NAME=text-embedding-ada-002
+AZURE_OPENAI_PROMPT_DEPLOYMENT_NAME=gpt-35-turbo
+AZURE_OPENAI_PROMPT_MODEL_NAME=gpt-35-turbo
+AZURE_OPENAI_DEFAULT_ENDPOINT=https://griptape-cloud-nonprod-scus.openai.azure.com/
```

Comment:
Lets keep internal endpoint env var values like this not in the .env.example, treat them like secrets (even though they aren't)